AppCenter.podspec

```diff
@@ -21,6 +21,9 @@ Pod::Spec.new do |s|
                       4. App Center Push (iOS and macOS):
                       App Center Push enables you to send push notifications to users of your app from the App Center portal. You can also segment your user base based on a set of properties and send them targeted notifications.
 
+                      5. App Center DataStorage (iOS):
+                      TODO: Add description for Datastorage
```

Comment:
We created a work item to update the description when we'll have the right wording.